cmd/mesh/profile-common.go

```diff
@@ -141,8 +150,30 @@ func genICPS(inFilename, profile, setOverlayYAML, ver string, force bool, l *Log
 	return finalYAML, finalICPS, nil
 }
 
-func genProfile(helmValues bool, inFilename, profile, setOverlayYAML, configPath string, force bool, l *Logger) (string, error) {
-	finalYAML, finalICPS, err := genICPS(inFilename, profile, setOverlayYAML, "", force, l)
+func getClusterSpecificValues(config *rest.Config) string {
```

Comment:
Optional: I suggest restructuring this function a bit:
- create a function to test for JWT support e.g. j, err := clusterSupportsThirdPartyJWT()
- use MakeTreeFromSetList([]string{fmt.Sprintf("values.global.sds.useTrustworthyJwt=%t", j)})